app/models/stripe_record.rb

```diff
@@ -76,11 +80,39 @@ def update_status(api_transaction)
   def retrieve_stripe
     case self.stripe_record_type
     when 'payment_intent'
-      Stripe::PaymentIntent.retrieve(self.stripe_id, stripe_account: self.find_account_id)
+      Stripe::PaymentIntent.retrieve(self.stripe_id, stripe_account: self.account_id)
     when 'charge'
-      Stripe::Charge.retrieve(self.stripe_id, stripe_account: self.find_account_id)
+      Stripe::Charge.retrieve(self.stripe_id, stripe_account: self.account_id)
     when 'refund'
-      Stripe::Refund.retrieve(self.stripe_id, stripe_account: self.find_account_id)
+      Stripe::Refund.retrieve(self.stripe_id, stripe_account: self.account_id)
+    end
+  end
+
+  alias_method :retrieve_remote, :retrieve_stripe
+
+  def update_amount_remote(amount_iso, currency_iso)
```

Comment:
Do we want to change this to `update_amount_stripe` for consistency with `retrieve_stripe`?

Reply:
I was debating with that but ideally, we want to move this method out of `stripe_account` and into `payment_intent`, so that _any_ record can update its remote regardless of whether it's Stripe or PayPal or whatnot.